Osiris/Hacks/InventoryChanger.cpp

```diff
@@ -223,12 +226,20 @@ class StaticData {
                     _paintKits.emplace_back(0, L"");
                     _gameItems.emplace_back(Type::Skin, 6, item->getWeaponId(), _paintKits.size() - 1, image);
                 }
-            } else if (isCollectible) {
+            } 
+            else if (isCollectible) {
                 if (const auto image = item->getInventoryImage()) {
                     _collectibles.emplace_back(isOriginal);
                     _gameItems.emplace_back(Type::Collectible, item->getRarity(), item->getWeaponId(), _collectibles.size() - 1, image);
                 }
-            } else if (itemTypeName == "#CSGO_Tool_Name_TagTag") {
+            }
+            else if (item->getCapabilities() & ITEM_CAP_CAN_PATCH) {
+                if (const auto image = item->getInventoryImage()) {
+                    _paintKits.emplace_back(0, L"");
+                    _gameItems.emplace_back(Type::Agent, item->getRarity(), item->getWeaponId(), _paintKits.size() - 1, image);
```

Comment:
`_paintKits.emplace_back(0, L"");` is for adding "vanilla" paint kits. Agents do not use paint kits at all so remove that line and replace `_paintKits.size() - 1` with zero.